drivers/osdp/src/osdp.c

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2020 Siddharth Chandrasekaran <siddharth@embedjournal.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <init.h>
+#include <device.h>
+#include <drivers/uart.h>
+#include <sys/ring_buffer.h>
+#include <logging/log.h>
+
+#include "osdp_common.h"
+
+LOG_MODULE_REGISTER(osdp, CONFIG_OSDP_LOG_LEVEL);
+
+struct osdp osdp_ctx;
```

Comment:
Shouldn't these three variables be static?

Reply:
Done.